src/storage/sqlite.py

```
@@ -7684,7 +7918,10 @@ def get_source_storage_stats(self, source_id: str) -> dict[str, Any]:
             "imf_vintages": ("indicator_vintages", "source = ?", ("imf",), "scraped_at"),
             "market": ("market_prices", "1 = 1", tuple(), "scraped_at"),
             "fed": ("central_bank_comms", "source = ?", ("fed",), "scraped_at"),
-            "calendar": ("calendar_events", "1 = 1", tuple(), "scraped_at"),
+            "calendar": (
+                "v_calendar_item", "1 = 1", tuple(),
```

Ruff (0.15.10)

[warning] Unnecessary `tuple()` call (rewrite as a literal)

Rewrite as a literal

(C408)